tarea 01

rclbox opened the pull request:
declaramos tres variables de diferente tipo y calculamos su promedio
Version 0.1

rclbox committed:
tarea 01

declaramos tres variables de diferente tipo y calculamos su promedio

rclbox commented:
Hello, this is my first homework I think I finally put it achieves this site. I hope to continue learning about the art of programming

cvhupci commented on hw_01_id_14.cpp:
```diff
@@ -1,19 +1,15 @@
+//en esta tarea se declararan tres tipos diferentes de variables y se calculara el promedio 
+#include<iostream>//biblioteca
+int main(){
+	int x=2;//declaramos un tipo de variable
```
Los tres tipos de datos debieran ser del mismo tipo

cvhupci commented on hw_01_id_14.cpp:
```diff
@@ -1,19 +1,15 @@
+	float y=3.9;//declaramos otro tipo de variable con decimales
+	double z=6.9;//declaramos otro tipo de variable doble
+	double p=(x+y+z)/3;//hacemos la operacion en una variable doble para calcular el promedio
+	std::cout<<"el promedio de los tres diferentes valores es:\n"<<p<<std::endl;//imprimimos el promedio
```
Sea amable con el usuario, imprima el valor de cada una de las variables x,y,z antes de imprimir la media

rclbox committed:
homework 01

tres valores para cada tipo de dato y a esos tres valores de esos tres
tipos de datos se les saca el promedio

rclbox commented:
aqui esta con tres datos, de tres diferentes tipos de variable
